pkg/client/cmd/config.go

```diff
@@ -159,6 +172,38 @@ func setCluster(cmd *cobra.Command, args []string) {
 	}
 }
 
+// rmCluster removes a server from the config file
+func rmCluster(cmd *cobra.Command, args []string) {
+	if len(args) == 0 {
+		cmd.Usage()
+		return
+	}
+
+	name := args[0]
+	if name == "" {
+		cmd.Usage()
+		return
+	}
+
+	c, err := client.ReadConfigFile(cfgFile)
+	if err != nil {
+		c = &client.Config{
+			Clusters:       make(map[string]client.ClusterConfig),
+			CurrentCluster: name,
+		}
+	}
```

Comment:
You guess you don't need to handle this case for rm-cluster, just PrintErrorAndExit.

Reply:
@aguerra good point. Something like this instead?

`client.PrintErrorAndExit("Error trying to read config file: %v", err)`